### PR TITLE
remove tags from hazard classes

### PR DESCRIPTION
**Pull request by emanuel-schmid**
Changes proposed in this PR:
- Remove anything dealing with tags from the hazard classes of climada_petals.
  This is part of the overall climada refactoring to remove the Tag classes.

This PR fixes #

### PR Author Checklist

- [ ] Read the [Contribution Guide][contrib]
- [x] Correct target branch selected (if unsure, select `develop`)
- [x] Descriptive pull request title added
- [x] Source branch up-to-date with target branch
- [ ] [Documentation](https://climada-python.readthedocs.io/en/latest/guide/Guide_PythonDos-n-Donts.html#Commenting-&-Documenting) updated
- [x] [Tests][testing] updated
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]
- [ ] [Changelog](https://github.com/CLIMADA-project/climada_petals/blob/main/CHANGELOG.md) updated

### PR Reviewer Checklist

- [ ] Read the [Contribution Guide][contrib]
- [ ] [CLIMADA Reviewer Checklist](https://climada-python.readthedocs.io/en/latest/guide/Guide_Reviewer_Checklist.html) passed
- [ ] [Tests][testing] passing
- [ ] No new [linter issues][linter]

[contrib]: https://github.com/CLIMADA-project/climada_python/blob/main/CONTRIBUTING.md
[testing]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html
[linter]: https://climada-python.readthedocs.io/en/latest/guide/Guide_Continuous_Integration_and_Testing.html#Static-Code-Analysis

### CODE REVIEW

File: climada_petals/hazard/low_flow.py [hunk 1]
```diff
@@ -40,7 +40,6 @@
 from climada.hazard.base import Hazard
 from climada.hazard.centroids import Centroids
 import climada.util.coordinates as u_coord
-from climada.util.tag import Tag
```
Comment (peanutfun) [inline climada_petals/hazard/low_flow.py:43]: How did this work before? I don't see a `climada/util/tag.py` module in any of the recent releases

File: climada_petals/hazard/low_flow.py [hunk 2]
```diff
@@ -272,15 +271,11 @@ def from_netcdf(cls, input_dir=None, centroids=None, countries=None, reg=None,
...
-                                        yearrange, yearrange_ref, gh_model, cl_model,
-                                        scenario, scenario_ref, soc, soc_ref, fn_str_var, keep_dis_data)
+                                        yearrange, keep_dis_data)
         return haz
 
     def set_intensity_from_clusters(self, centroids=None, min_intensity=1, min_number_cells=1,
-                                    yearrange=TARGET_YEARRANGE, yearrange_ref=REFERENCE_YEARRANGE,
-                                    gh_model=None, cl_model=None,
-                                    scenario='historical', scenario_ref='historical', soc='histsoc',
-                                    soc_ref='histsoc', fn_str_var=FN_STR_VAR, keep_dis_data=False):
```
Comment (peanutfun) [inline climada_petals/hazard/low_flow.py:283]: Curious, that all of this information was only put into the tag 🤔

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+
 - shapely `1.8` -> `2.0` ([#80](https://github.com/CLIMADA-project/climada_petals/pull/80))
 
 ### Added
 
 ### Changed
 
 - refactored `climada_petals.river_flood.RiverFlood.from_nc`, removing calls to `set_raster` ([#80](https://github.com/CLIMADA-project/climada_petals/pull/80))
+- several adaptations have been necessary because of the `Tag` class being removed from the CLIMADA core package:
+  [#88](https://github.com/CLIMADA-project/climada_petals/pull/88)
```
Comment (peanutfun) [inline CHANGELOG.md:21]: I suggest to put this into the "Removed" category:

```
### Removed

- `tag` attribute from hazard classes [#88](https://github.com/CLIMADA-project/climada_petals/pull/88)
```

**Review by peanutfun — APPROVED** [review]
Looks good! I only suggest to improve the Changelog entry.

File: climada_petals/hazard/low_flow.py (hunk 1 above)
Comment (emanuel-schmid) [reply climada_petals/hazard/low_flow.py:43]: after release 3.3, hazard.tag and entity.tag have been merged

**Comment by emanuel-schmid** [comment]
🙌 thanks for the review!

**Pull request merged**